docs/_sass/_main.scss

```diff
@@ -0,0 +1,19 @@
+@charset 'utf-8';
+@import url('https://fonts.googleapis.com/css?family=Open+Sans:300,400,600,700&display=swap');
+
+$family-sans-serif: 'Open Sans', sans-serif;
+
+// Color Variables
+$primary: #2092d2 !default;
+$secondary: #2fb39b !default;
+$gray: #999;
```

Comment:
I find the contrast on the text in the "What is the DSA" section a bit low.  Can we use a somewhat darker grey here?

Reply:
Fixed in 27f408b8d0311cdf44401bbdd023452f144ef19f